main_test.go

```diff
@@ -65,7 +65,29 @@ func initTestSpec() {
 	applicationFeeRefundCreateMethod = &spec.Operation{}
 	applicationFeeRefundGetMethod = &spec.Operation{}
 
-	chargeAllMethod = &spec.Operation{}
+	chargeAllMethod = &spec.Operation{
+		Parameters: []*spec.Parameter{
+			{
+				In:       spec.ParameterQuery,
+				Name:     "limit",
+				Required: false,
+				Schema: &spec.Schema{
+					Type: spec.TypeInteger,
+				},
+			},
+		},
+		Responses: map[spec.StatusCode]spec.Response{
+			"200": {
+				Content: map[string]spec.MediaType{
+					"application/json": {
+						Schema: &spec.Schema{
+							Type: spec.TypeObject,
+						},
+					},
+				},
+			},
+		},
+	}
```

Comment:
This new detail in the fixture API routes is needed to get the new `TestStubServer_QueryParam*` tests below working (we need an endpoint that'll take query string parameters to test against).